waspc/src/Wasp/Generator.hs

```diff
@@ -55,9 +56,6 @@ genApp spec =
     <++> genDb spec
     <++> genDockerFiles spec
 
-checkGeneratedCode :: AppSpec -> Path' Abs (Dir ProjectRootDir) -> IO [GeneratorWarning]
-checkGeneratedCode spec dstDir = DbGenerator.checkGeneratedCode spec dstDir
```

Comment:
Removed the level of indirection (`DbGenerator.checkGeneratedCode`) and called the underlying function (`DbGenerator.warnIfDbSchemaChangedSinceLastMigration`) directly in `postGenDbGeneratorActions `.